adserver/utils.py

```diff
@@ -216,6 +225,21 @@ def get_geolocation(ip_address):
     return None
 
 
+def build_blocked_ip_set():
+    """Build a set of blocked IPs for preventing bogus ad impressions."""
+    blocked_ips = set()
+
+    filepath = os.path.join(settings.GEOIP_PATH, "torbulkexitlist")
```

Comment:
This file could be a setting. Also, it could be a setting with a list of filenames so we can extend later.

Reply:
Currently, I think this is unlikely to be extended in that way especially since that would require changes to the `Makefile` and other sources may not come in the same format. Long term, I think we will buy a downloadable database for this.